default.nix

```diff
@@ -358,152 +269,138 @@ let overrideCabal = pkg: f: if pkg == null then null else haskellLib.overrideCab
           sha256 = "13i6lz99x0jb9fgns7brlqnv5s5w4clp26l8c3kxd318r1krvr6w";
         }) {};
 
-        superconstraints =
-          # Remove override when assertion fails
-          assert (super.superconstraints or null) == null;
-          self.callPackage (self.haskellSrc2nix {
-            name = "superconstraints";
-            src = fetchurl {
-              url = "https://hackage.haskell.org/package/superconstraints-0.0.1/superconstraints.cabal";
-              sha256 = "0bgc8ldml3533522gp1x2bjiazllknslpl2rvdkd1k1zfdbh3g9m";
-            };
-            sha256 = "1gx9p9i5jli91dnvvrc30j04h1v2m3d71i8sxli6qrhplq5y63dk";
-          }) {};
-      } // (if enableLibraryProfiling then {
-        mkDerivation = expr: super.mkDerivation (expr // { enableLibraryProfiling = true; });
-      } else {});
+        mkDerivation = expr: super.mkDerivation (expr // {
+          inherit enableLibraryProfiling;
```

Comment:
Profiling is enabled by default now in libraries. Given that, do we even want this to be configurable anymore?